### PR TITLE
Fix include from ParsedownTest

### PR DESCRIPTION
**Pull request by greut**
I wasn't able to run all the tests from ParsedownExtra because of it.

### CODE REVIEW

**Comment by erusev** [comment]
@hkdobrev do you have any thoughts on this?

File: test/ParsedownTest.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ public function test_no_markup()
 
     public function testLateStaticBinding()
     {
-        include 'test/TestParsedown.php';
+        include dirname(__file__) . '/TestParsedown.php';
```
Comment (hkdobrev) [inline test/ParsedownTest.php]: I think we could just use `__DIR__` instead of `dirname(__FILE__)` as Parsedown no longer supports PHP 5.2.

**Comment by hkdobrev** [comment]
@erusev I don't have anything else to add other than the comment above. Seems legit.

@greut Could you elaborate more on what problem you have experienced when running parsedown-extra tests? Probably something has changed as it wasn't a problem before?

**Comment by greut** [comment]
```
PHPUnit 5.6.2 by Sebastian Bergmann and contributors.

...........................................................E      60 / 60 (100%)

Time: 53 ms, Memory: 6.00MB

There was 1 error:

1) ParsedownExtraTest::testLateStaticBinding
include(test/TestParsedown.php): failed to open stream: No such file or directory

/srv/http/pouille2/vendor/erusev/parsedown/test/ParsedownTest.php:142
/srv/http/pouille2/vendor/erusev/parsedown/test/ParsedownTest.php:142

ERRORS!
Tests: 60, Assertions: 59, Errors: 1.
```

**Comment by greut** [comment]
The problem occurs when I'm running the parsedown-extra tests from the parsedown-extra directory in the case where `parsedown` is the parent (sibling).
https://github.com/erusev/parsedown-extra/blob/master/test/bootstrap.php#L3-L5

```
erusev/parsedown
erusev/parsedown-extra
```

**Comment by greut** [comment]
Thanks!

**Pull request merged**